boards/arm/weact_stm32g431_core/weact_stm32g431_core_defconfig

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2023 Andreas Sandberg
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_SOC_SERIES_STM32G4X=y
+CONFIG_SOC_STM32G431XX=y
+
+CONFIG_CLOCK_CONTROL=y
+CONFIG_PINCTRL=y
+
+CONFIG_ARM_MPU=y
+CONFIG_HW_STACK_PROTECTION=y
+
+CONFIG_GPIO=y
+CONFIG_LED=y
```

Comment:
CONFIG_LED shouldn't be enabled as part of default board configuration.

Reply:
Done.